test/chat-room.test.js

```diff
@@ -0,0 +1,66 @@
+/*eslint-disable no-console*/
+
+const { ChatRoom } = require('../lib/chat-room');
+
+describe('ChatRoom', () => {
```

Comment:
To avoid some duplication, you could add a beforeEach() that would be something like: 
```suggestion
 beforeEach(() => {
        const newChat = new Clients();
    });

```